container/dangerzone.py

```diff
@@ -228,8 +238,9 @@ async def document_to_pixels(self) -> None:
             args = [
                 "libreoffice",
                 "--headless",
+                "--safe-mode",
```

Comment:
> * We can start `libreoffice` with the `--safe-mode` flag, which I think (again, in my system) opens documents with Macro Security Level High.

This is one of the cases where terminology can be confusing. Here they're talking about safety and not security. This option exists for [recovering from borked instalations](https://help.libreoffice.org/latest/en-US/text/shared/optionen/macrosecurity_sl.html). So it won't actually do what we want.

Reply:
I'm OK with keeping it. But in that case can tweak a bit the last part of the commit? It says something like this:


> * We preemptively run LibreOffice in safe mode, to remove hardware
  acceleration and make sure **that macros are not invoked as well.**

Reply:
Maybe split out that part of the commit since it's not about supporting more doc formats?

Reply:
Sure, I can split it and explain the bold part better.

Reply:
Done in 58a8241844d6625667144b442ef95dc5b20e9e46.